make/prepare

```diff
@@ -337,7 +400,23 @@ if customize_crt == 'on' and openssl_installed():
     empty_subj = "/C=/ST=/L=/O=/CN=/"
     private_key_pem = os.path.join(config_dir, "ui", "private_key.pem")
     root_crt = os.path.join(config_dir, "registry", "root.crt")
-    create_root_cert(empty_subj, key_path=private_key_pem, cert_path=root_crt)
+    if args.ha_mode:
+        shared_private_key_pem = os.path.join(base_dir, "ha", "private_key.pem")
```

Comment:
a little confused with the naming of the variables, sometime it's "common_xxx" sometime it's "share_xxxx".

Also don't quite understand the purpose to copy files back and forth between `config/` and `ha/` directories.

Let's discuss it.